client/components/replacePost/index.jsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable no-underscore-dangle */
+/* global React */
+
+import PostSelector from '../postSelector';
+
+const {
+  apiFetch,
+  components: {
+    Spinner,
+  },
+  data: {
+    select,
+    useDispatch,
+  },
+  element: {
+    useState,
+    useEffect,
+  },
+  i18n: {
+    __,
+  },
+} = wp;
```

Comment:
See comment above about importing from `@wordpress`. The `useState` and `useEffect` imports should come from `react`.